src/doom/r_main.c

```diff
@@ -699,23 +700,43 @@ void R_InitLightTables (void)
    // [crispy] smooth diminishing lighting
     if (crispy->smoothlight)
     {
-	LIGHTLEVELS = 32;
-	LIGHTSEGSHIFT = 3;
-	LIGHTBRIGHT = 2;
-	MAXLIGHTSCALE = 48;
-	LIGHTSCALESHIFT = 12;
-	MAXLIGHTZ = 1024;
-	LIGHTZSHIFT = 17;
+#ifdef CRISPY_TRUECOLOR
+    if (crispy->truecolor)
+    {
+	    // [crispy] if in TrueColor mode, use smoothest diminished lighting
+	    NUMCOLORMAPS =     32 << 3;
```

Comment:
Do we even have to set `NUMCOLORMAPS` again in `R_InitLightTables()`? It is already set in `R_InitColormaps()`.

Reply:
I apologize again. It's a fourth day of having a 🤧, and third time of having a 🤧 in two weeks. Was able to quickly recover first two times, but third one finally got me. The best thing I can do is to follow orders and instructions, rather than thinking clearly by myself.

Will commit requested changes shortly.

Reply:
> Do we even have to set NUMCOLORMAPS again in R_InitLightTables()? It is already set in R_InitColormaps()

This is a bit tricky, as we handing `NUMCOLORMAPS` exclusively for TrueColor in `R_InitColormaps()`, while in `R_InitLightTables()` we handling all possible cases. Having `NUMCOLORMAPS = 32 << 3;` is redundant there, but it gives some "sync" with other conditions below. What is your recommendation then? Just delete it?

Reply:
> but it gives some "sync" with other conditions below. 

I'd rather see it as a potential source for bugs. 😉 

Let's explicitly set `NUMCOLORMAPS` also in the `#ifndef CRISPY_TRUECOLOR` branch of `R_InitColormaps()` and not touch it again in `R_InitLightTables()`.